art/art_dic.py

```diff
@@ -468,5 +468,6 @@
     "surprised2": "(O_o)",
     "boombox2": "♫♪ |̲̅̅●̲̅̅|̲̅̅=̲̅̅|̲̅̅●̲̅̅| ♫♪",
     "equalizer": "▇ ▅ █ ▅ ▇ ▂ ▃ ▁ ▁ ▅ ▃ ▅ ▅ ▄ ▅ ▇",
+    "playing cards": "[♥]]] [♦]]] [♣]]] [♠]]]"
```

Comment:
can we divide this art into 4 art?
like `"playing cards hearts" : "[♥]]]"`

Reply:
Good point ;-)
I added 12 new arts for individual mode https://github.com/sepandhaghighi/art/pull/92/commits/90e5ab4664efa8d8b242b6a4748490e80e0b17f4 https://github.com/sepandhaghighi/art/pull/92/commits/8bb2a4c1266f6aaeb4963118155de3b8e670db9b
I also found [this](https://en.wikipedia.org/wiki/Playing_cards_in_Unicode) set, but I think  these playing cards are too small !!

Reply:
I checked the link, i think these are interesting sets and worth adding, i didn't catch your point about smallness of these sets.
if it is OK for i can add this link to [Font and Art suggestion](https://github.com/sepandhaghighi/art/issues/59)

Reply:
> I checked the link, i think these are interesting sets and worth adding, i didn't catch your point about smallness of these sets.
> if it is OK for i can add this link to [Font and Art suggestion](https://github.com/sepandhaghighi/art/issues/59)

Okay, It's up to you ;-)